assessments/0002-notary-project.md

```diff
@@ -0,0 +1,176 @@
+
+### Recommendations
+
+V1 Docs will likely need to be reworked once an information architecture is setup for the new site. The new site IA can likely be informed by the existing V1 docs.
+
+The notaryproject.dev site should primarily focus on the V2 work, but V1 needs to be represented as well.
+- V1 content seems quite good, but looks like it was developed for a site that no longer uses it.
+- For MVP, I'd suggest focusing on V2 content, and linking out to the existing V1 documentation in the location that is currently rather than incorporating it right off the bat.
+
+Eventually I'd expect that V1 would be moved to the Notary Project org, but that is a project level consideration. Even if the whole repo doesn't get moved, there may be value in moving the docs out of TUF's notary repo and into the notaryproject/notaryproject.dev repo.
+
+## Contributor documentation 
+
+**Note**, as my recommendation above is to link to the V1 documentation as is for the website MVP, the remainder of this assessment will only be looking at the existing V2 documentation.
+
+Criteria 
+- 1 = (Is not present or requires significant work)
+- 3 = (is present, but needs work)
+- 5 = (is executed extremely well or no improvement required) |
+
+| Criteria                                  | 1   | 2   | 3   | 4   | 5   |
+| ---                                       | --- | --- | --- | --- | --- |
+| Communication methods documented          |     |     | ✅  |     |     |
+| Beginner friendly issue backlog           |     | ✅  |     |     |     |
+| “New contributor” getting started content | ✅  |     |     |     |     |
+| Project governance documentation          | ✅  |     |     |     |     |
+
+### Comments
+
+_Provide comments for each rating above, 1-2 sentences max, bullet point list_
+
+**Communication methods documented**
+- There is a Contributing & Conversations section in the main README (tho, it is mis-linked in the table of contents)
+
+**Beginner friendly issue backlog**
+- There are "Documentation," "spec," and "good first issue" issue tags
+- Most issues seem to link out to hackmd.io notes, but those note are are well written and informative
+
+**“New contributor” getting started content**
+- Same as above, there is a Contributing & Conversations section, but there are not yet any instructions for contributions.
+
+**Project governance documentation**
+- This appears to be actively being worked on, and there is a "Governance" issue tag in use. 
+
+
+### Recommendations
+
+Most of the issues in this section are a function of not having a website yet, and the project still being in a building phase.
+- As we go through the site build project, we should put together a skeleton IA that can be filled in as the project grows.
+
+Regarding beginner friendliness, as we move through the project, we should consider processes/documents for:
```

Comment:
```suggestion
Regarding beginner friendliness, consider processes/documents for:
```

In this document, I wouldn't use "we" - please keep in mind that you are there to advise the project and help out with some specific tasks, _not_ to become a permanent member of the project team.